src/control/server/mgmt_client.go

```diff
@@ -59,6 +59,26 @@ func newMgmtSvcClient(ctx context.Context, log logging.Logger, cfg mgmtSvcClient
 	}
 }
 
+// delayRetry delays next retry.
+func (msc *mgmtSvcClient) delayRetry(ctx context.Context) {
+	delayCtx, cancelDelayCtx := context.WithTimeout(ctx, retryDelay)
+	select {
+	case <-delayCtx.Done():
+	}
+	cancelDelayCtx()
+}
+
+func (msc *mgmtSvcClient) logAttempt(prefix string, err error, status *int32) {
+	switch {
+	case err != nil:
+		msc.log.Debugf(prefix+": %v", err)
```

Comment:
Nit: The mixture of string concatenation and format directives is a little odd. Recommend picking one or the other.